src/json.hpp.re2c

```diff
@@ -6168,6 +6168,21 @@ class basic_json
               thousands_sep(!loc->thousands_sep ? '\0' : loc->thousands_sep[0]),
               decimal_point(!loc->decimal_point ? '\0' : loc->decimal_point[0])
         {}
+        serializer(const serializer& cpy)
```

Comment:
Ok, but I still think we should `= default ` all operations if we're going down that road.

It's an empty struct after all. Also, nobody ever declares a variable of type `std::is_same<int, void> pointless;`.

However `pointless ` is still copyable, moveable etc.

This might be nit-picking, but I don't see a good reason to delete those constructors.